src/localization/utils/load-crowdin-strings.ts

```
@@ -7,6 +7,7 @@ export async function loadCrowdinStrings(
   fallbackStrings?: object,
   prefix?: string,
 ): Promise<void> {
+  debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)